drivers/dht/dht.c

```diff
@@ -178,33 +184,23 @@ int dht_read(dht_t *dev, int16_t *temp, int16_t *hum)
         _reset(dev);
 
         /* validate the checksum */
-        uint8_t sum = (raw_temp >> 8) + (raw_temp & 0xff) + (raw_hum >> 8)
-                    + (raw_hum & 0xff);
+        uint8_t sum = (raw_temp_I) + (raw_temp_D) + (raw_hum_I) + (raw_hum_D);
         if (sum != csum) {
             DEBUG("error: checksum doesn't match\n");
             return DHT_NOCSUM;
         }
 
         /* parse the RAW values */
-        DEBUG("RAW values: temp: %7i hum: %7i\n", (int)raw_temp, (int)raw_hum);
-        switch (dev->params.type) {
-            case DHT11:
-                dev->last_val.temperature = (int16_t)((raw_temp >> 8) * 10);
-                dev->last_val.humidity = (int16_t)((raw_hum >> 8) * 10);
-                break;
-            /* DHT21 == DHT22 (same value in enum), so both are handled here */
-            case DHT22:
-                dev->last_val.humidity = (int16_t)raw_hum;
-                /* if the high-bit is set, the value is negative */
-                if (raw_temp & 0x8000) {
-                    dev->last_val.temperature = (int16_t)((raw_temp & ~0x8000) * -1);
-                }
-                else {
-                    dev->last_val.temperature = (int16_t)raw_temp;
-                }
-                break;
-            default:
-                return DHT_NODEV;      /* this should never be reached */
+        DEBUG("RAW values: temp: %2i.%i hum: %2i.%i\n", (int)raw_temp_I,
```

Comment:
```suggestion
        DEBUG("[dht] RAW values: temp: %2i.%i hum: %2i.%i\n", (int)raw_temp_I,
```

prefixing debug messages makes it easier to distinguish between them, when debugging issues affecting multiple modules

Reply:
Done (Also on other DEBUGs)